Dockerfile

```diff
@@ -67,3 +67,9 @@ RUN mkdir /opt/rwd/ && \
     | tar -xzC /opt/rwd --strip-components=1 && \
     cd /opt/rwd && \
     pip install --no-cache-dir -r requirements.txt
+
+RUN pip install flask==0.10.1
```

Comment:
I tried adding `RUN pip install -r /opt/rwd/src/api/requirements.txt` but it doesn't work because it's pulling rwd from the develop branch, and that file isn't there (since it's only been added at the moment on my branch). Leaving things as is (running `pip install flask` in Dockerfile) works for now, but it seems like we need a better solution for building container images in development vs. production mode.